### PR TITLE
[somfytahoma] fix the smoke sensor's documentation

### PR DESCRIPTION
**Pull request by octa22**
This PR fixes the documentation for smoke sensor's beep tests mentioned in https://github.com/openhab/openhab-addons/issues/12858

Signed-off-by: Ondrej Pecta <opecta@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties [hunk 1]
```diff
@@ -99,8 +99,8 @@ channel-type.somfytahoma.active_zones_state.label = Active Zones State
 channel-type.somfytahoma.active_zones_state.description = Active Zones State
 channel-type.somfytahoma.alarm_check.label = Alarm Check Trigger
 channel-type.somfytahoma.alarm_check.description = A channel for triggering the smoke sensor's check
```
Comment (lolodomo) [inline bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties:101]: beep check

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties (hunk 1 above)
Comment (octa22) [reply bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties]: fixed

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Fix #12858

**Pull request merged**